assets/reader-activation-auth/auth-form.js

```diff
@@ -162,40 +170,28 @@ window.newspackRAS.push( function ( readerActivation ) {
 						body.set( 'reader-activation-auth-form', 1 );
 						body.set( 'npe', emailInput.value );
 						body.set( 'action', 'link' );
-						readerActivation
-							.getCaptchaV3Token() // Get a token for reCAPTCHA v3, if needed.
-							.then( captchaToken => {
-								if ( ! captchaToken ) {
-									return;
-								}
-								body.set( 'captcha_token', captchaToken );
+
+						fetch( form.getAttribute( 'action' ) || window.location.pathname, {
+							method: 'POST',
+							headers: {
+								Accept: 'application/json',
+							},
+							body,
+						} )
+							.then( () => {
+								form.setMessageContent( newspack_reader_activation_labels.code_resent );
+								container.setFormAction( 'otp' );
+								readerActivation.setOTPTimer();
 							} )
 							.catch( e => {
-								console.log( { e } ); // eslint-disable-line no-console
+								console.log( e ); // eslint-disable-line no-console
```

Comment:
Might be a good time to remove this `console.log`. Is it necessary? If the purpose is for debugging, I think we can always re-add this on our end temporarily while working through any issues.

Reply:
39c4e67e removes the `console.log` and implements error handling in case the request to send an OTP email fails. Currently the method on the backend [always returns JSON](https://github.com/Automattic/newspack-plugin/blob/fix/recaptcha-for-ras-acc/includes/reader-activation/class-reader-activation.php#L1526) so that's why the error isn't caught by the `catch` block.